DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -30,12 +30,12 @@
                                        DQMMessageLogger )
 
 from DQMOffline.Muon.muonMonitors_cff import *
-diMuonHistos.etaBin = cms.int32(70) #dimuonhistograms mass, bin
-diMuonHistos.etaBBin = cms.int32(70)
-diMuonHistos.etaEBin = cms.int32(70)
-diMuonHistos.etaBinLM = cms.int32(12)
-diMuonHistos.etaBBinLM = cms.int32(12)
-diMuonHistos.etaEBinLM = cms.int32(12)
+diMuonHistos.etaBin = cms.int32(175) #dimuonhistograms mass, bin
```

Comment:
@goni - please move these changes into the CFI itself (module parameters should only be changed for clones)